src/pages/gen1/[platform]/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -739,6 +739,74 @@ function App() {
 </Block>
 </BlockSwitcher>
 
+### Redirect URLs
+
+You can provide multiple _Sign in Redirect URI(s)_ & _Sign out redirect URI(s)_.
+
+If you have multiple redirect URI inputs, you'll need to pass them in your Amplify configuration. For example:
+
+```javascript
+Amplify.configure({
+  Auth: {
+    Cognito: {
+      loginWith: {
+        oauth: {
+          redirectSignIn: [
+            'myDevApp://'
+          ],
+          redirectSignOut: [
+            'myDevApp://',
+            'myProdApp://'
+          ],
+          ...oauthConfig
+        }
+      },
+      ...userPoolConfig
+    }
+  }
+});
+```
+
+#### Option to specify a redirect URL upon signOut
```

Comment:
I think this section can be moved to the signOut API section on gen2 docs instead. The changes we are making are affecting the redirect experience during signOut.

Reply:
Not opposed to that but Gen2 has no section on signInWithRedirect so there is no push to use that API on the Gen2 docs yet and this is particularly only useful with HostedUI in picture. Putting it in the general signOut page is unnecessary without context on signInWithRedirect.

Reply:
Discussed offline, added it to Gen2 docs